src/actions/appActions.js

```diff
@@ -268,6 +269,10 @@ export const setupSentryAction = (user: ?Object, wallet: Object) => {
         ethAddress: address,
       },
     });
+    // eslint-disable-next-line i18next/no-literal-string
+    Instabug.setUserAttribute('address', wallet.address);
```

Comment:
```suggestion
    Instabug.setUserAttribute('address', address);
```

We already pulled out `address` above - we can just use that instead